src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (firstVar) => {
+    count++;
+
+    const currentCallNumber = count;
+
+    if (currentCallNumber < 3 || currentCallNumber % 2 === 1) {
+      return (secondVar) => {
+        return firstVar + secondVar;
```

Comment:
You can use short arrow notation, for this type of return

Reply:
fixed